src/client/fuse_ll.cc

```diff
@@ -175,13 +180,13 @@ class CephFuse::Handle {
 
 static int already_fuse_mounted(const char *path, bool &already_mounted)
 {
+  int err = 0;
+  already_mounted = false;
+#if defined(__linux__)
```

Comment:
Well, I meant something like this:

```
#if defined(__linux__)
static int already_fuse_mounted(const char *path, bool &already_mounted) {
...
...
}
# else // non-linux platforms
static int already_fuse_mounted(const char *path, bool &already_mounted) {
...
...
}
#endif
```
Just that its a _tiny_ bit easier to read.

Reply:
@vshankar Done.